account_move_template/i18n/es.po

```diff
@@ -0,0 +1,547 @@
+msgstr "Última actualización en"
+
+#. module: account_move_template
+#: model:ir.model.fields,field_description:account_move_template.field_account_move_template__line_ids
+#: model:ir.model.fields,field_description:account_move_template.field_account_move_template_run__line_ids
+msgid "Lines"
+msgstr "Líneas"
+
+#. module: account_move_template
+#: model:ir.model.fields,field_description:account_move_template.field_account_move_template_line__template_id
+msgid "Move Template"
+msgstr "Plantilla de asiento"
+
+#. module: account_move_template
+#: model:ir.model.fields,field_description:account_move_template.field_account_move_template__name
+#: model:ir.model.fields,field_description:account_move_template.field_account_move_template_line_run__name
+msgid "Name"
+msgstr "Nombre"
+
+#. module: account_move_template
+#: model_terms:ir.ui.view,arch_db:account_move_template.account_move_template_run_form
+msgid "Next"
+msgstr "Siguiente"
+
+#. module: account_move_template
+#: model:ir.model.fields,field_description:account_move_template.field_account_move_template_line__note
+#: model:ir.model.fields,field_description:account_move_template.field_account_move_template_line_run__note
+msgid "Note"
+msgstr "Nota"
+
+#. module: account_move_template
+#: model:ir.model.fields,field_description:account_move_template.field_account_move_template_line__tax_line_id
+#: model:ir.model.fields,field_description:account_move_template.field_account_move_template_line_run__tax_line_id
+msgid "Originator Tax"
+msgstr "Impuesto origen"
+
+#. module: account_move_template
+#: model:ir.model.fields,field_description:account_move_template.field_account_move_template_run__partner_id
+msgid "Override Partner"
+msgstr "Sobreescribir Contacto"
+
+#. module: account_move_template
+#: model:ir.model.fields,field_description:account_move_template.field_account_move_template_run__overwrite
+msgid "Overwrite"
+msgstr "Sobreescribir"
+
+#. module: account_move_template
+#: code:addons/account_move_template/wizard/account_move_template_run.py:0
+#, python-format
+msgid "Overwrite value must be a valid python dict"
+msgstr "El valor de sobreescritura debe de ser un diccionario python válido"
+
+#. module: account_move_template
+#: model:ir.model.fields,field_description:account_move_template.field_account_move_template_line__partner_id
+#: model:ir.model.fields,field_description:account_move_template.field_account_move_template_line_run__partner_id
+msgid "Partner"
+msgstr "Empresa"
+
+#. module: account_move_template
+#: model:ir.model.fields,field_description:account_move_template.field_account_move_template_line__payment_term_id
+#: model:ir.model.fields,field_description:account_move_template.field_account_move_template_line_run__payment_term_id
+msgid "Payment Terms"
+msgstr "Términos de pago"
+
+#. module: account_move_template
+#: model:ir.model.fields,field_description:account_move_template.field_account_move_template_line__python_code
+msgid "Python Code"
```

Comment:
There are commits in the branch 16.0 that are missing here, e.g. f9ebf94026bd8a2f227771ebd3ff319aaa4937b1